rust/src/router/data_consumer.rs

```diff
@@ -903,7 +903,7 @@ impl DataConsumer {
     }
 
     /// Callback is called when the associated data producer is resumed.
-    pub fn on_producer_resume<F: Fn() + Send + Sync + 'static>(&self, callback: F) -> HandlerId {
+    pub fn on_data_producer_resume<F: Fn() + Send + Sync + 'static>(&self, callback: F) -> HandlerId {
```

Comment:
This is a breaking change BTW. Either we should make a minor version bump or keep the old name alongside the new name.

Reply:
I'm pretty sure nobody noticed it. But we can release next version later. Will do.

Reply:
I know, but is that this is a breaking change that will potentially make code fail to compile, we should honor that

Reply:
See https://github.com/versatica/mediasoup/pull/1271/commits/b36e52556e55d69907edbd0da999e1f37346e463

Reply:
https://github.com/versatica/mediasoup/pull/1272